js/main.js

```diff
@@ -19,29 +17,14 @@ $(document).ready( function() {
 });
 
 // This is a conditional (Ternary) Operator. It assigns a value based on some other value
-// It is much shorter to write than if-else.
-// How it works:
-// 		localStorage allows you to store values in the client side. Its like cookies
-//		but they don't get send to the server
-//		1. If 'visited' is already defined:
-//			a. get the 'visited' and convert it to a number (+) since
-//			   localStorage stores it as a string
-//			b. Add +1 to it
-//		2. If 'visited' is not defined (null), set it to 1, since you are on the website
-//	   	   right now.
-//		3. Assign the value you chose back to the 'visited' in localStorage
 localStorage['visited'] = localStorage['visited'] ? +localStorage['visited']+1 : 1;
 
-// Assign 'visited' to variable v, because its shorter to write, and will not require
-// I/O every time you use it, the above was aleady consuming...
-const v = +localStorage['visited'];
```

Comment:
I actually think this is over engineered and we should just decide what color to show based on a `math.Random`